src/components/TasksTable/index.jsx

```diff
@@ -88,6 +106,9 @@ export default class TasksTable extends Component {
 
       return filteredItems
         .filter(item => !tag || item.tags.includes(tag))
+        .filter(
+          item => !project || project === 'All' || item.project === project
+        )
```

Comment:
Is there a way we can avoid looping on the list twice and perform both filters in one filter method?